packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -701,321 +673,6 @@ export class UnfurlService extends BaseService {
         );
     }
 
-    private async saveScreenshot({
```

Comment:
I'd pin this commit somewhere, in case we need to compare playwright config against some old logic in puppeteer

Reply:
https://github.com/lightdash/lightdash/releases/tag/before-playwright-puppeteer-replacement